CHANGELOG.md

```diff
@@ -47,6 +47,7 @@
 - Generate distribution dependencies catalog [#8987](https://github.com/hyperledger/besu/pull/8987)
 - Add `WorldStateService` to the plugin API [#9024](https://github.com/hyperledger/besu/pull/9024)
 - Wait for peers before starting Backward Sync [#9003](https://github.com/hyperledger/besu/pull/9003)
+- LUKSO Mainnet Pectra Hardfork [#9070](https://github.com/hyperledger/besu/pull/9070)
```

Comment:
this has just missed the 25.8.0 release so will need to move to the unreleased section. There will be another release before 17 Sept

Reply:
@Wolmin I can't update your branch so you'll need to make this change and merge latest main

Reply:
@macfarla updated, waiting for CI/CD now :+1:

Reply:
@macfarla should I remove this from `CHANGELOG.md`?